extensions/interactions/NumericInput/protractor.js

```diff
@@ -32,6 +32,8 @@ var expectInteractionDetailsToMatch = function(elem) {
 var submitAnswer = function(elem, answer) {
   elem.element(by.tagName('oppia-interactive-numeric-input')).
     element(by.tagName('input')).sendKeys(answer + '\n');
```

Comment:
OK, I'll try what you suggested. Enter used to work before I removed the ng-form I guess.